### PR TITLE
Add xmovie

### PR DESCRIPTION
**Pull request by jbusecke**
## PR Summary

Adding xmovie based on twitter request (https://twitter.com/story645/status/1453454851933941772?s=20)

### CODE REVIEW

File: packages/xmovie.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ repo: jbusecke/xmovie
 section: animations
 description: A simple way of creating beautiful movies from xarray objects.
 site: # optional, default repo site
```
Comment (jklymak) [inline packages/xmovie.yml:4]: I guess we shouldn't have these comments in here.  Just delete any empty lines I think.  Sorry for the bother!

Comment (jbusecke) [reply packages/xmovie.yml]: Ahh, thanks for the clarification. Some explanation in the comments would have helped a lot indeed.

**Review by timhoffm — APPROVED** [review]

**Comment by timhoffm** [comment]
Thanks @jbusecke!

File: packages/xmovie.yml (hunk 1 above)
Comment (jklymak) [reply packages/xmovie.yml]: I think I fixed this so folks won't have this problem again....

**Pull request merged**